src/Console/Installer.php

```diff
@@ -156,7 +156,17 @@ public static function setFolderPermissions($dir, $io)
             }
         };
 
-        $worldWritable = bindec('0000000111');
+        $worldWritable = 007;
+        if ($worldWritable === 007) {
```

Comment:
Okay. **Please simply close this PR if you deem it to be unnecessary,** and don't see it - or a variant of it - as an improvement over the status quo.

I went into figuring out why composer install set the other flag to read-write-execute running `composer install` on deployed applications to update their dependencies. I am on IRC for further questions.

TBH the permission flags should only be set when the project is created by composer the first time not every time someone does an update.